stylus/ui-base/fonts.styl

```diff
@@ -20,6 +20,10 @@ $font-title
 $font-labor
     font-family  Lato, sans-serif
 
+$font-smoothing
+    -moz-osx-font-smoothing grayscale
```

Comment:
Well, either the font-smoothing is only relevant with Lato font and in that case just add this in the `$font-labor` placeholder, or it's for any font and in that case, just directly write it in body declaration in `default.styl`.

See what i mean?